cli/config/single_or_array_test.go

```diff
@@ -0,0 +1,252 @@
+package config_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/tt/cli/config"
+	"gopkg.in/yaml.v3"
+)
+
+type singleOrArrayCase[T any] struct {
+	name     string
+	data     []byte
+	expected config.SingleOrArray[T]
+	wantErr  bool
+}
+
+func testSingleOrArrayJSON[T any](t *testing.T, tests []singleOrArrayCase[T]) {
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// t.Parallel()
```

Comment:
Maybe, you forgot to remove this?

Reply:
Thanks, it was commented out just for debugging.